tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -172,6 +175,10 @@ def parse_args():
                              "number of nodes. The saved optimizations can further optimize nodes not assigned to the "
                              "compiling EP at runtime.")
 
+    parser.add_argument('--optimization_level', default='all', choices=['disable', 'basic', 'extended', 'all'],
+                        help='Graph optimization level to use. '
+                             'For general usage, the default of \'all\' should be fine.')
+
```

Comment:
hey just for better understanding, I thought we removed `optimization_level` option previously, adding it back here for?

Reply:
Being able to specify the optimization level is useful for generating the test model added in this PR. I guess it could be useful if one wanted to examine models at a particular optimization level too. But I expect the default value is generally fine.

Reply:
I'm wary about adding an explicit option that has no specific use case for an external user and potentially adds confusion around how to use the conversion script. 

Maybe we could leave the changes to pass through an optimization level but require that to be manually overridden by changing the script locally.

Reply:
ok, changed it to being settable with an environment variable not externally documented instead.